src/stub.rs

```diff
@@ -14,9 +14,11 @@ pub use stub_config::StubConfig;
 pub fn generate(config: StubConfig, generator: &str) -> Result<String> {
     let mut stub = parser::parse_generator_stub(generator)?;
 
-    for processor in config.language.preprocessors.iter() {
-        processor(&mut stub);
-    }
+    let processor = config
+        .language
+        .preprocessor
+        .expect("a valid deserialized preprocessor should not be None");
+    processor(&mut stub);
```

Comment:
This breaks stubs for any language without a preprocessor:
```
» cargo run gen ruby
   Compiling coctus v0.2.0 (/home/fourhundredlux/Code/github/ellnix/coctus)
    Finished dev [unoptimized + debuginfo] target(s) in 6.91s
     Running `target/debug/coctus gen ruby`
thread 'main' panicked at src/stub.rs:20:10:
a valid deserialized preprocessor should not be None
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
Instead I suggest:
```suggestion
    if let Some(processor) = config.language.preprocessor {
        processor(&mut stub)
    }
```